samples/assistant/README.md

```diff
@@ -38,7 +45,7 @@ The assistant will invoke a skill function whenever it decides to do so to satis
 
 The sample is available in the following language stacks:
 
-* [C# on the in-process worker](csharp-inproc)
+* [C# on the out-of-process worker](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=windows)
 
 Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
 
```

Comment:
I can't add a comment further down, but we need to make several other updates to this README. A few examples:
* Replace `csharp-inproc` with `csharp-ooproc` on line 60 and anywhere else it may appear.
* Remove all mention of the entity trigger (there are at least two mentions that need to be removed, lines 78 and 81)
* Update the JSON output from lines 150-203 (I think it's just some of the top level properties that we changed?)
* We should mention somewhere in here where we store the chat history and how this can be configured.

Reply:
Updated!